src/skeleplex/data/tubes.py

```diff
@@ -0,0 +1,62 @@
+"""Drawing tubes in a volume as example data."""
+
+import numpy as np
+from skimage.draw import disk
+
+
+def draw_tube(image, center=(50, 50), diameter=20, z_span=(25, 75)):
```

Comment:
please add type annotations for all functions. they help the IDE check the usage of the function (e.g., are the inputs the correct type) and we will use them when building the API docs. Here's a small intro on the python type hints/annotations: https://fastapi.tiangolo.com/python-types/#simple-types

```suggestion
def draw_tube(image: np.ndarray, center: tuple[int, int] = (50, 50), diameter:int =20, z_span: tuple[int, int]=(25, 75)):
```